runtime-modules/content/src/permissions/mod.rs

```diff
@@ -1,3 +1,20 @@
+/*
+* The following table summarizes the permissions in the content subsystem.
+* - Actor role as columns, controller account is Tx sender.
+* - operations on a given channel (=channel=) are rows, which are basically the guards to be
+*   implemented
+* - Entries are conditions to be verified / assertions
+*
+* |                       | *Lead*                   | *Curator*                | *Member*                | *Collaborator*                   |
+* |-----------------------+--------------------------+--------------------------+-------------------------+----------------------------------|
+* | *assets mgmt*         | channel.owner is curator | curator is channel.owner | member is channel.owner | collaborators in channel.collabs |
+* | *censorship mgmt*     | channel.owner is curator | curator is channel.owner | false                   | false                            |
+* | *category mgmt*       | true                     | true                     | false                   | false                            |
+* | *collab. set mgmt*    | channel.owner is curator | curator is channel.owner | member is channel.owner | false                            |
+* | *reward account mgmt* | false                    | curator is channel.owner | member is channel.owner | false                            |
+* | *create channel*      | false                    | true                     | true                    | false                            |
+* | *delete channel*      | false                    | curator is channel.owner | member is channel.owner | false                            |
```

Comment:
I'm not sure why the lead is not authorized to delete channel or update reward account using `Lead` context, while it is authorized to update assets or collaborators using this context. Previously I believe the lead could act as owner of any curator channel (which I think made sense, as (s)he is the one managing the curator groups